addons/im_livechat/static/src/core/web/thread_actions.js

```diff
@@ -3,17 +3,49 @@ import { threadActionsRegistry } from "@mail/core/common/thread_actions";
 import { _t } from "@web/core/l10n/translation";
 import { LivechatChannelInfoList } from "@im_livechat/core/web/livechat_channel_info_list";
 
-threadActionsRegistry.add("livechat-info", {
-    component: LivechatChannelInfoList,
-    condition(component) {
-        return component.thread?.channel_type === "livechat";
-    },
-    panelOuterClass: "o-livechat-ChannelInfoList bg-inherit",
-    icon: "fa fa-fw fa-info",
-    iconLarge: "fa fa-fw fa-lg fa-info",
-    name: _t("Information"),
-    nameActive: _t("Close Information"),
-    sequence: 10,
-    sequenceGroup: 7,
-    toggle: true,
-});
+threadActionsRegistry
+    .add("livechat-info", {
+        component: LivechatChannelInfoList,
+        condition(component) {
+            return component.thread?.channel_type === "livechat";
+        },
+        panelOuterClass: "o-livechat-ChannelInfoList bg-inherit",
+        icon: "fa fa-fw fa-info",
+        iconLarge: "fa fa-fw fa-lg fa-info",
+        name: _t("Information"),
+        nameActive: _t("Close Information"),
+        sequence: 10,
+        sequenceGroup: 7,
+        toggle: true,
+    })
+    .add("livechat-status", {
+        component: LivechatChannelInfoList,
+        condition(component) {
+            return component.thread?.channel_type === "livechat";
+        },
```

Comment:
It should only be displayed for non-closed channels:
See condition on panel: `!props.thread.livechat_end_dt or props.thread.livechat_status`
Note: I've actually made them exclusive in SQL, we might only keep the `!props.thread.livechat_end_dt` condition now.

Reply:
Wait maybe the comment is not on the right line, I'm talking about the "quick set status" from discuss sidebar.